hypershift-operator/controllers/nodepool/metrics/metrics.go

```diff
@@ -28,6 +31,10 @@ const (
 	CountByHClusterMetricName = "hypershift_hostedcluster_nodepools" // What about renaming it to hypershift_cluster_nodepools ?
 	countByHClusterMetricHelp = "Number of NodePools for a given HostedCluster"
 
+	CoresCountByHClusterMetricName = "hypershift_cluster_cores"
+	coresCountByHClusterMetricHelp = "Number of cores for a given HostedCluster. " +
+		"-1 if this number cannot be computed." // Be careful when changing this metric as it is used for billing the customers
```

Comment:
Why would billing use this instead of using the metrics already generated in the data plane? node_role_os_version_machine:cpu_capacity_cores:sum{_id=….,  label_node_role_kubernetes_io="worker"}

Reply:
Forgot to say, but there is not much documentation besides what can be found in the epic:
https://issues.redhat.com/browse/SDE-3517